dotnet/Microsoft.MacCatalyst.Templates/maccatalyst/Entitlements.plist

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If you remove this file you shouldn't need any provisioning profiles.

Reply:
Should `Entitlements.plist` be in the template at all?

Reply:
should we have a template for one ? (cross platform)
IIRC the IDE do not create one "on demand", the file must exists for the editor to work

Reply:
I don't think so, since it makes a provisioning profile required.

For iOS it's different, because the user must always sign an app for device, but that's not necessary for macOS.

Reply:
I think it's still prompting me for a provisioning profile, here is a build log: [catalyst.zip](https://github.com/xamarin/xamarin-macios/files/6062349/catalyst.zip)

I made an iOS app `com.companyname.foo` in Xcode to get it to create a profile for me, but that one isn't picked up either.

Reply:
OK, that's a bug, I found at least one cause (which will be fixed in a different PR).